.github/workflows/ci.yml

```diff
@@ -35,9 +35,18 @@ jobs:
       - name: Build docker image
         run: make docker
 
-      - name: Install meshnet
+      - name: Install meshnet with VXLAN link
         run: make install
 
       - name: Run tests
         run: make e2e
 
+      - name: Uninstall meshnet with VXLAN link
+        run: make uninstall
+
+      - name: Install meshnet with GRPC link
+        run: make grpc=1 install
+
```

Comment:
Did you measure how long it takes to run this in GH actions? would it make sense to split this into a separate job and run it VXLAN and GRPC in parallel?

Reply:
Actions don't run by default on forked repos. We'll look into it. But I do agree, these two could be separate jobs.